contrib/TestHarness/Program.cs

```diff
@@ -477,7 +480,7 @@ static int Run(string fdbserverName, string tlsPluginFile, string testFolder, st
                     memCheckThread.Join();
                     consoleThread.Join();
 
-                    var traceFiles = Directory.GetFiles(tempPath, "trace*.xml");
+                    var traceFiles = Directory.GetFiles(tempPath, "trace*.*").Where(s => s.EndsWith(".xml") || s.EndsWith(".json")).ToArray();

```

Comment:
This line has been changed from the initial version.  Limiting to `SearchOption.TopDirectoryOnly` was not appropriate.